servant/src/Servant/API/Redirect.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# OPTIONS_HADDOCK not-home    #-}
+module Servant.API.Redirect (Redirect (..)) where
+
+import           Data.Typeable
+                 (Typeable)
+import           Prelude ()
+import           Prelude.Compat
+import           Network.URI
+                 (URI (..))
+
+-- | TODO: TBW...
+-- 
+-- TODO: Add to ComprehensiveAPI
+newtype Redirect = Redirect URI deriving (Typeable, Eq, Ord, Show)
```

Comment:
I think this should take a status code as well (since there are multiple types of redirect) or a type indicating one of the status codes.

Reply:
Indeed. Will do.